xml/ay_bigfile.xml

```diff
@@ -17082,16 +17078,16 @@ mv /var/run/zypp.sav /var/run/zypp.pid
    <qandaentry xml:id="qa.ay.insecure">
     <question>
      <para>
-      linuxrc blocks the installation with <literal>File not
+      <command>Linuxrc</command> blocks the installation with <literal>File not
       signed</literal>. I need to manually interact.
      </para>
     </question>
     <answer>
      <para>
-      Linuxrc found some unsigned file (like a driver update). To
-      use an unsigned file, you can suppress that message by passing
-      <literal>insecure=1</literal> to the linuxrc parameter list (together
-      with the <literal>autoyast=...</literal> parameter).
+      <command>Linuxrc</command> found an unsigned file, such as a driver
```

Comment:
If tagged as 'command', it should be spelled/spelt in lowercase, even if it is located at the beginning of a sentence.